src/components/Auth/UsersCard/table/UsersCleared.test.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
```

Comment:
This bit shouldn't be necessary anymore, after react 17 or so

Reply:
I get this error on the `<UsersCleared />` component when I remove the import: `'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.`
 
I'm not entirely sure why importing React is necessary here, but it seems to break the build when I remove it.